src/components/Title.jsx

```diff
@@ -0,0 +1,6 @@
+import style from './Title.module.css';
+
+function Title({ children, size }) {
+  return <h1 className={style[size]}>{children}</h1>;
+}
```

Comment:
Is it the only option for a title? How many h1 with the same look do expect on a page?

Again, try to see how UI libraries solve this component to make it more generic.